.github/workflows/build-agent.yml

```diff
@@ -34,12 +34,18 @@ jobs:
           node-version: 20
           registry-url: 'https://registry.npmjs.org'
 
+      # See: https://github.com/actions/cache/blob/5a3ec84eff668545956fd18022155c47e93e2684/examples.md#node---npm
+      - name: Get npm cache directory
+        id: npm-cache-dir
+        shell: pwsh
+        run: echo "dir=$(npm config get cache)" >> ${env:GITHUB_OUTPUT}
```

Comment:
Neat.  I didn't think this would work on Windows, but I guess PowerShell ftw?  👍

Reply:
Yeah powershell makes Windows a lot less bad, sometimes 😛